docs: consolidate `core22`+ graphics docs, and add `core26` ones (#282)
MirServer/ubuntu-frame: docs/how-to/use-snap-graphics.md

(use-snap-graphics)=

(the-graphics-core22-snap-interface)=

(the-gpu-2404-snap-interface)=

# How to use graphics in snaps

> This document describes how to use the Snap interfaces supplying graphics drivers, what are the requirements to create a content provider snap as well as discusses the design of the interface.

Reviewer: ```suggestion
This document describes how to use the Snap interfaces supplying graphics drivers, what the requirements are to create a content provider snap, and the design of the interface.
```
